.github/workflows/test-template.yml

```diff
@@ -189,7 +189,7 @@ jobs:
   # CLIENT (functional tests with mini-proxy)
   #
   client-miniproxy:
-    name: Client (functional tests with mini-proxy)
+    name: Client (func/mini-proxy)
```

Comment:
the `/` here indicates that this is a subset of all functional tests?
or are these different from what is called `func` in some other jobs?